api/types/accesslist/accesslist.go

```diff
@@ -258,6 +261,12 @@ type Grants struct {
 	Traits trait.Traits `json:"traits" yaml:"traits"`
 }
 
+// AccessListStatus contains dynamic fields calculated during retrieval.
+type AccessListStatus struct {
+	// MemberCount is the number of members in the access list.
+	MemberCount *int
```

Comment:
Why not match the size of the number from the proto? What does using `int` get us?

Reply:
Good point. I'll adjust this to uint32.